app/res/layout/audio_prototype.xml

```diff
@@ -1,67 +1,86 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:layout_gravity="center"
-        android:gravity="center"
-        android:layout_marginTop="10dp">
+        android:layout_marginTop="10dp"
+        android:orientation="horizontal">
 
         <ImageButton
+            android:id="@+id/play_audio"
             android:layout_width="65dp"
             android:layout_height="65dp"
-            android:background="@drawable/play_disabled"
-            android:id="@+id/play_audio"
+            android:layout_marginEnd="10dp"
             android:layout_marginRight="10dp"
-            android:elevation="10dp"
-            android:layout_marginEnd="10dp" />
+            android:background="@drawable/play_disabled"
+            android:elevation="10dp" />
+
+        <SeekBar
+            android:id="@+id/seekBar"
+            android:layout_width="match_parent"
+            android:layout_height="65dp" />
+
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="right"
+        android:layout_marginTop="10dp"
+        android:orientation="horizontal">
+
+        <TextView
+            android:id="@+id/playback_time"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            tools:text="00:20" />
 
         <TextView
+            android:id="@+id/playback_duration"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:textColor="@color/grey"
-            android:id="@+id/recording_text"/>
+            tools:text="/00:30" />
 
     </LinearLayout>
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
         android:layout_gravity="center"
+        android:layout_marginTop="30dp"
         android:gravity="center"
-        android:layout_marginTop="10dp">
+        android:orientation="horizontal">
 
         <ImageButton
+            android:id="@+id/capture_button"
             android:layout_width="80dp"
             android:layout_height="80dp"
-            android:background="@drawable/record"
-            android:id="@+id/capture_button"
+            android:layout_marginEnd="30dp"
             android:layout_marginRight="30dp"
-            android:elevation="10dp"
-            android:layout_marginEnd="30dp" />
+            android:background="@drawable/record"
+            android:elevation="10dp" />
 
         <ImageButton
+            android:id="@+id/choose_file"
             android:layout_width="80dp"
             android:layout_height="80dp"
             android:background="@drawable/open_file"
-            android:id="@+id/choose_file"
-            android:elevation="10dp"/>
+            android:elevation="10dp" />
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/2aa03aecc11ade0381c1f426b782e030115c4729/app/res/layout/audio_prototype.xml#L81"></span>Attribute <code>elevation</code> is only used in API level 21 and higher (current min is 16)</td>
    </tr>
  </tbody>
</table>